io/net/bonding.py

```diff
@@ -286,9 +288,12 @@ def bond_fail(self, arg1):
                                 interface %s" % interface)
             time.sleep(self.sleep_time)
 
-        bond_mtu = ['2000', '3000', '4000', '5000', '6000', '7000',
-                    '8000', '9000']
-        for mtu in bond_mtu:
+        if self.physical_interface:
+            self.bond_mtu = ['2000', '3000', '4000', '5000', '6000', '7000',
```

Comment:
I don't think we need to create a class variable `self.bond_mtu` as we are only using that array in this context (context being the bond_fail function). 

Other than that, LGTM!

Reply:
I think you're going to want to have two bond_mtu list variables. One for the host interface and one for the peer interface as you can have a virtual interface sending traffic to a physical and vice versa.